ui/dashboards/src/components/PanelDrawer/PanelDrawer.tsx

```diff
@@ -95,7 +101,7 @@ export const PanelDrawer = () => {
               </Button>
             </Stack>
           </Box>
-          <PanelEditorForm initialValues={panelEditor.initialValues} onChange={handleChange} />
+          <PanelEditorForm mode={panelEditor.mode} initialValues={panelEditor.initialValues} onChange={handleChange} />
```

Comment:
I don't love having to drill this prop (`mode`). The issue that I ran into is that `usePanelEditor` lives in `dashboards` folder. However, `<PanelSpecEditor>` lives in `plugin-system` folder, and I am pretty sure we don't import from `dashboards` into `plugin-system` (`dashboards` can pull from `plugin-system`, but not vice versa). 

Relates to an overall thing where editing a panel is currently pretty tightly coupled with a panel being part of a dashboard. If we de-couple this more in the future, should be easier to use `usePanelEditor` in other places.

Reply:
Yeah decoupling more in the future sounds great, but like you said, can’t import from dashboards in plugin-system w/o creating a circular dependency.

Sometimes you just gotsta to drill those props 🤷‍♂️